embedchain/migrations/versions/40a327b3debd_create_initial_migrations.py

```diff
@@ -0,0 +1,62 @@
+"""Create initial migrations
+
+Revision ID: 40a327b3debd
+Revises:
+Create Date: 2024-02-18 15:29:19.409064
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "40a327b3debd"
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
```

Comment:
Getting an exception when starting my bot:
`  File "/mnt/c/Users/KārlisBalcers/Documents/MeaChat/.venv/lib/python3.10/site-packages/embedchain/migrations/versions/40a327b3debd_create_initial_migrations.py", line 23, in upgrade
    op.create_table(
  File "<string>", line 8, in create_table
  File "<string>", line 3, in create_table
  File "/mnt/c/Users/KārlisBalcers/Documents/MeaChat/.venv/lib/python3.10/site-packages/alembic/operations/ops.py", line 1311, in create_table
    return operations.invoke(op)
  File "/mnt/c/Users/KārlisBalcers/Documents/MeaChat/.venv/lib/python3.10/site-packages/alembic/operations/base.py", line 445, in invoke
    return fn(self, operation)
  File "/mnt/c/Users/KārlisBalcers/Documents/MeaChat/.venv/lib/python3.10/site-packages/alembic/operations/toimpl.py", line 131, in create_table
    operations.impl.create_table(table)
  File "/mnt/c/Users/KārlisBalcers/Documents/MeaChat/.venv/lib/python3.10/site-packages/alembic/ddl/impl.py", line 366, in create_table
    self._exec(schema.CreateTable(table))
  File "/mnt/c/Users/KārlisBalcers/Documents/MeaChat/.venv/lib/python3.10/site-packages/alembic/ddl/impl.py", line 207, in _exec
    return conn.execute(construct, multiparams)
  File "/mnt/c/Users/KārlisBalcers/Documents/MeaChat/.venv/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1408, in execute
    return meth(
  File "/mnt/c/Users/KārlisBalcers/Documents/MeaChat/.venv/lib/python3.10/site-packages/sqlalchemy/sql/ddl.py", line 180, in _execute_on_connection
    return connection._execute_ddl(
  File "/mnt/c/Users/KārlisBalcers/Documents/MeaChat/.venv/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1519, in _execute_ddl
    ret = self._execute_context(
  File "/mnt/c/Users/KārlisBalcers/Documents/MeaChat/.venv/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1839, in _execute_context
    return self._exec_single_context(
  File "/mnt/c/Users/KārlisBalcers/Documents/MeaChat/.venv/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1979, in _exec_single_context
    self._handle_dbapi_exception(
  File "/mnt/c/Users/KārlisBalcers/Documents/MeaChat/.venv/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 2335, in _handle_dbapi_exception
    raise sqlalchemy_exception.with_traceback(exc_info[2]) from e
  File "/mnt/c/Users/KārlisBalcers/Documents/MeaChat/.venv/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1960, in _exec_single_context
    self.dialect.do_execute(
  File "/mnt/c/Users/KārlisBalcers/Documents/MeaChat/.venv/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 924, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) table ec_chat_history already exists
[SQL: 
CREATE TABLE ec_chat_history (
        app_id VARCHAR NOT NULL, 
        id VARCHAR NOT NULL, 
        session_id VARCHAR NOT NULL, 
        question TEXT, 
        answer TEXT, 
        metadata TEXT, 
        created_at TIMESTAMP, 
        PRIMARY KEY (app_id, id, session_id)
)

]`

Reply:
Same, this release caused a regression.